dash/frontend/src/app/modules/private/pages/cluster/kubesec/kubesec.component.ts

```diff
@@ -176,12 +176,16 @@ export class KubesecComponent implements OnInit, OnDestroy {
   decideScoreColor(score: string): string {
     const scoreNum = +score;
     if (scoreNum <= 0) {
+      // red
       return this.scoreColors[0];
```

Comment:
Could always change scoreColors to an object: 

var scoreColors = {
  red: '#FF0000',
  etc
}